why-conventional-commits.qmd

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Why conventional commits (with optional emojis)"
-description: "Our reasons for using conventional commits (with optional emojis following the Gitmoji convention) across Seedcase projects."
+description: "Our reasons for using conventional commits (with optional emojis following the Gitmoji convention) across Seedcase codebases."
```

Comment:
Here and below just trying to avoid using "projects" when we have "the Seedcase Project" as an overarching category.